src/api/api.js

```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+
+export function makeApi() {
+  return axios.create({
+    baseURL: "https://rickandmortyapi.com/api/",
+  });
+}
+
+// La utiilidad está en que de estqa forma podemos hacer:
+// axios.get("/episodes") y ya lo tenemos, sin tener que poner la baseURL
```

Comment:
Be careful with commenting lines in different languages, just for cleanliness. Although I find it great that you comment them!